### PR TITLE
[Bug 1254690] Fix escaping of share link on mobile articles

### PR DESCRIPTION
**Pull request by mythmon**
This blatantly steals the same solution used in #2804.

@Osmose r?

### CODE REVIEW

File: kitsune/wiki/tests/test_templates.py [hunk 1]
```diff
@@ -438,6 +438,16 @@ def test_document_view(self):
         eq_(200, response.status_code)
         assert template_used(response, 'wiki/mobile/document.html')
 
+    def test_document_share_link_escape(self):
+        """Ensure that the share link isn't escaped."""
+        r = ApprovedRevisionFactory(
+            content='Test',
+            document__share_link='https://www.example.org',
+        )
+        response = self.client.get(r.document.get_absolute_url())
```
Comment (safwanrahman) [inline kitsune/wiki/tests/test_templates.py]: is it going fetching the mobile page template? how?

Comment (safwanrahman) [inline kitsune/wiki/tests/test_templates.py]: **"going to fetch"

Comment (mythmon) [inline kitsune/wiki/tests/test_templates.py]: Yes, it will, since this is a MobileTestCase, which makes sure that the mobile templates are used.

**Comment by safwanrahman** [comment]
strongly believe this should get r+

**Comment by Osmose** [comment]
Late r+

**Pull request merged**